main.js

```diff
@@ -119,19 +119,12 @@ function startApp() {
   }
 
   function handleUnhandledException(err) {
-    bt.report(err, {}, handleFinishedReport);
+    Sentry.captureException(err);
+    handleFinishedReport();
```

Comment:
We also need to change the dialog that pops up.  The copy is really bad.  I can do that in a separate PR.

Reply:
This was done